appeals/web/src/server/lib/mappers/appeal/submappers/ip-comments-due-date.mapper.js

```diff
@@ -1,31 +1,18 @@
 import { dateISOStringToDisplayDate } from '#lib/dates.js';
+import { textSummaryListItem } from '#lib/mappers/components/text.js';
 
 /** @type {import('../appeal.mapper.js').SubMapper} */
-export const mapIpCommentsDueDate = ({ appealDetails, currentRoute }) => ({
-	id: 'ip-comments-due-date',
-	display: {
-		summaryListItem: {
-			key: {
-				text: 'Comments from interested parties due'
-			},
-			value: {
-				// @ts-ignore
-				html: dateISOStringToDisplayDate(appealDetails.appealTimetable?.ipCommentsDueDate) || ''
-			},
-			actions: {
-				items: [
-					appealDetails.validAt
-						? {
-								text: 'Change',
-								href: `${currentRoute}/appellant-case/valid/date`,
-								visuallyHiddenText:
-									'The date all case documentation was received and the appeal was valid',
-								attributes: { 'data-cy': 'change-ip-comments-due-date' }
-						  }
-						: {}
-				]
-			},
-			classes: 'appeal-ip-comments-due-date'
-		}
-	}
-});
+export const mapIpCommentsDueDate = ({
+	appealDetails,
+	currentRoute,
+	userHasUpdateCasePermission
+}) =>
+	textSummaryListItem({
+		id: 'ip-comments-due-date',
+		text: 'Comments from interested parties due',
+		// @ts-ignore
```

Comment:
Why is ts-ignore being used here?

Reply:
One to pick up on separate types work I think. This was there already.

`Property 'ipCommentsDueDate' does not exist on type 'WebAppealTimetable'.`

Reply:
yeah that was me originally, afaik that data just wasn't/isn't in the db schema yet